### PR TITLE
add filter list to configuration

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos

### CODE REVIEW

File: dashboard/test/unit/test_cbroker.js [hunk 1]
```diff
@@ -253,5 +254,25 @@ suite('cbroker', function () {
 
     });
 
+    test('should_filter_region_list_and_remove_disabled_region', function () {
```
Comment (flopezag) [inline dashboard/test/unit/test_cbroker.js:257]: Just some comments here.
- A disabled region will have correct data in the CB, although it was obsoleted.
- Why do you add in the expected the 'Region5' without data?

Comment (jesuspg) [inline dashboard/test/unit/test_cbroker.js:257]: this data only is a sample data. The input data not is deterministic for the new functionality. You could see the same in other test: (e.g. should_return_a_json_with_all_regions_and_status).

**Comment by flopezag** [comment]
Just to fix the travis error...

**Comment by jframos** [comment]
Coverage decreased?

File: dashboard/test/unit/test_cbroker.js (hunk 1 above)
Comment (flopezag) [inline dashboard/test/unit/test_cbroker.js:257]: NTC

**Comment by jframos** [comment]
LGTM

**Pull request merged**